src/matcher.py

```diff
@@ -17,48 +16,48 @@ def matcher_lambda_handler(event, lambda_context):
     print(event)
     outputs = []
     if "Records" in event:
-        sqs_client = boto3.client("sqs")
         s3_client = boto3.client("s3")
-        for record in event["Records"]:
-            message = json.loads(record['body'])['Message']
-            s3_records = json.loads(message)['Records']
-            for s3_record in s3_records:
-                s3 = s3_record["s3"]
-                bucket = s3["bucket"]["name"]
-
-                key = urllib.parse.unquote(s3["object"]["key"])
-                logger.info("Object %s found", key)
-                logger.info("Bucket " + bucket)
-                s3_object = s3_client.get_object(Bucket=bucket, Key=key)
-                streaming_body = s3_object["Body"]
+        sqs_client = boto3.client("sqs")
+        rules = yara.load("output")
+        efs_root_location = os.environ["ROOT_DIRECTORY"]
+        for record in event['Records']:
+            message_body = json.loads(record['body'])
 
-                rules = yara.load("output")
-                match = rules.match(data=streaming_body.read())
+            for efs_download_record in message_body:
```

Comment:
You don't need this loop now. You can just use `message_body` in the place of `efs_download_record`